developer/src/inst/build.sh

```diff
@@ -20,9 +20,10 @@ builder_describe_outputs \
 
 builder_parse "$@"
 
-. "$KEYMAN_ROOT/resources/build/win/environment.inc.sh"
-. "$KEYMAN_ROOT/resources/build/win/wix.inc.sh"
-. "$KEYMAN_ROOT/resources/build/win/zip.inc.sh"
+. "${KEYMAN_ROOT}/resources/build/win/environment.inc.sh"
+. "${KEYMAN_ROOT}/resources/build/win/wix.inc.sh"
+. "${KEYMAN_ROOT}/resources/build/win/zip.inc.sh"
```

Comment:
Doesn't this line need to go?

Reply:
I had that first, but it turns out the current script uses two different ways of creating a zip file. #14338 will change that and will also remove that line.